concent_api/core/tests/test_payments_bankster.py

```diff
@@ -372,3 +376,44 @@ def test_that_settle_overdue_acceptances_should_return_claim_deposit_info_with_a
 
         self.assertEqual(requestors_claim_payment_info.amount_paid, 1000)
         self.assertEqual(requestors_claim_payment_info.amount_pending, 1000)
+
+
+class DiscardClaimBanksterTest(ConcentIntegrationTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.task_to_compute = self.task_to_compute = self._get_deserialized_task_to_compute()
```

Comment:
Why here is double `self.task_to_compute`?

Reply:
:+1: done